pampulha/apps/anamnesis/serializer.py

```diff
@@ -143,3 +157,39 @@ def validate_phone_number(self, phone_number: str) -> str:
             return phone_number
 
         return f"55{phone_number}"
+
+    def validate_age(self, age: int) -> int:
+        if age < 0:
+            raise ValidationError("Age cannot be less than 0.")
+
+        return age
+
+    def validate_father(self, father: str) -> str:
+        if " " not in father:
```

Comment:
quebra o nome e sobrenome com slice, ele vira uma lista, onde no indice 0 vai o nome, no indice 1 o sobrenome (caso tenha) caso nao tenha o len vai ser 1
```suggestion
        full_name = father.slice(" ")
        if len(fullname) <1:
```